kotest-property/src/jvmMain/kotlin/io/kotest/property/arbitrary/dates.kt

```
@@ -56,6 +56,7 @@ fun Arb.Companion.localDate(
    minDate: LocalDate = LocalDate.of(1970, 1, 1),
    maxDate: LocalDate = LocalDate.of(2030, 12, 31)
 ): Arb<LocalDate> {
+   if (minDate == maxDate) return Arb.constant(minDate)
```

Also i just realised maybe we should require maxdate >= mindate to fail early..